src/lib.rs

```diff
@@ -50,6 +50,44 @@ impl Error for ErrorKind {
     }
 }
 
+/// Enumeration of possible methods to seek within an I/O object.
+///
+/// Semantics are the same as [`std::io::SeekFrom`], check its documentation for details.
+#[derive(Debug, Copy, Clone, Eq, PartialEq)]
+#[cfg_attr(feature = "defmt", derive(defmt::Format))]
+pub enum SeekFrom {
+    /// Sets the offset to the provided number of bytes.
+    Start(u64),
+    /// Sets the offset to the size of this object plus the specified number of bytes.
+    End(i64),
+    /// Sets the offset to the current position plus the specified number of bytes.
+    Current(i64),
+}
+
+#[cfg(feature = "std")]
+#[cfg_attr(docsrs, doc(cfg(feature = "std")))]
+impl Into<std::io::SeekFrom> for SeekFrom {
```

Comment:
to implement conversions you have to impl `From`, not `Into` (`Into` is blanket-impl'd on `From`)

Reply:
Thanks, mis-remembered that.  Done.